.travis.yml

```diff
@@ -3,5 +3,6 @@ sudo: false
 before_script:
   - python -m SimpleHTTPServer > /dev/null 2>&1 &
   - sleep 2
```

Comment:
that is true, since it is backgrounded you need to wait a tiny bit for the python server to boot on up

:goat: :video_game: